### DISCUSSION TITLE
position of TODO checkbox

### DISCUSSION THREAD

**Discussion by danraymond** [NicolasPetton/Notebook#17] [Q&A]
Nico;

See below: Not sure if is a notebook or projectify issue.

The position of the check box is below the other toolbar items. I have tried rearranging in the control panel view toolbar settings, but it doesn't work too well, see second image where in the toolbar i moved it to just before the close tiddler one. Seems to be a wrap issue.

The TW is version 5.1.24 pre release. It's not happening in the 5.1.23 release. Have attached the TW.

![2021-01-07 11_37_51-Dans Base 20200104 TiddlyWiki — a non-linear personal web notebook — Mozilla Fir](https://user-images.githubusercontent.com/1908612/103840242-d6b2c780-50dc-11eb-9fa8-14b5ddd5b21a.png)

![2021-01-07 11_41_20-Dans Base 20200104 TiddlyWiki — a non-linear personal web notebook — Mozilla Fir](https://user-images.githubusercontent.com/1908612/103840436-4cb72e80-50dd-11eb-8175-0a73bd0728e6.png)

Tiddly wiki: [empty5124pre.zip](https://github.com/NicolasPetton/Notebook/files/5779034/empty5124pre.zip)

**Comment by NicolasPetton** [comment]
Thanks @danraymond for the feedback.

The bug seems to come from the Tekan plugin. It is styling all `label`s in the page, displaying them as block elements, which causes the issues you see.

Ping @ibnishak.
